lib/utils.js

```diff
@@ -0,0 +1,11 @@
+var path = require('path');
+
+/* Normalize URL or path to an absolute path, if it corresponds to one. */
+function filePath(urlOrPath) {
```

Comment:
I'm not sure any tool would output absolute file system paths by default?

If a source URL looked like `/source.js` then it is probably implied that it is rootURL relative.